crates/air/src/commands/format.rs

```diff
@@ -98,7 +98,11 @@ impl FormatMode {
 
 fn inform_changed(paths: &[PathBuf], f: &mut impl Write) -> io::Result<()> {
     for path in paths.iter().sorted_unstable() {
-        writeln!(f, "Would reformat: {}", path.display())?;
+        writeln!(
+            f,
+            "Would reformat: {path}",
+            path = relativize_path(path).underline()
+        )?;
```

Comment:
I made this one change to cli behavior so that snapshots are relative to the working directory and stable when run on CI.

In retrospect, it actually aligns with how we emit paths in error messages, so I think this is more correct anyways.